syntax/text/lexer/src/Luna/Syntax/Text/Lexer/Grammar.hs

```diff
@@ -6,7 +6,16 @@
 
 module Luna.Syntax.Text.Lexer.Grammar where
 
-import Prologue hiding (List, Type, Symbol, cons, span, range, catch, takeWhile, Text)
+import Prologue hiding
+    ( List
+    , Type
+    , Symbol
+    , cons
+    , span
+    , range
+    , catch
+    , takeWhile
+    , Text)
```

Comment:
Please use more complex form here, at least until we've got auto linter we can do this manually. 
Just
```
import Prologue hiding ( List, Type, Symbol, cons, span, range, catch
                       , takeWhile, Text)
```
etc